exploratory/citools/lasci_ominus1.py

```diff
@@ -21,6 +21,7 @@
 from mrh.my_pyscf.mcscf.lasci_sync import all_nonredundant_idx
 from mrh.my_pyscf.fci import csf_solver
 from itertools import product
+from memory_profiler import profile
```

Comment:
This adds a dependency that needs to be indicated in `mrh/.github/workflows/ci.yml`